.github/workflows/check-dist.yml

```diff
@@ -23,10 +23,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
```

Comment:
This will fail eventually being in v2.  It can be updated to v4. 
- actions/checkout@v4

Reply:
Thanks, I update actions/checkout newest version.